snprc_scheduler/src/org/labkey/snprc_scheduler/security/QCStateEnum.java

```diff
@@ -58,41 +67,35 @@ public void setPublicData(boolean publicData)
         _publicData = publicData;
     }
 
-    public int getValue()
+    public static Integer getValueByName (Container c, User u, QCStateEnum qcStateEnum)
```

Comment:
Are we expecting getValueByName() to return Integer?

Reply:
Yes, it returns an Integer.